apps/website/src/app/docs/report/page.tsx

```diff
@@ -109,75 +256,99 @@ export default function RatingShieldsPage() {
 
           <div className="flex items-center justify-center">
             <ReportBadge
-              name="lightwalker.eth"
-              data={getReportExampleData(Rating.pass)}
+              ensName={getExampleReportName(Rating.pass)}
+              data={getExampleReportData(Rating.pass)}
             />
           </div>
           <div className="flex items-center justify-center">
             <ReportBadge
-              name="culturecafé.eth"
-              data={getReportExampleData(Rating.warn)}
+              ensName={getExampleReportName(Rating.warn)}
+              data={getExampleReportData(Rating.warn)}
             />
           </div>
           <div className="flex items-center justify-center">
             <ReportBadge
-              name="‍420.eth"
-              data={getReportExampleData(Rating.alert)}
+              ensName={getExampleReportName(Rating.alert)}
+              data={getExampleReportData(Rating.alert)}
             />
           </div>
           <div className="flex items-center justify-center">
-            <ReportBadge name="notrab.eth" hadLoadingError={true} />
+            <ReportBadge
+              ensName={getExampleReportName(undefined, true)}
+              hadLoadingError={true}
+            />
           </div>
           <div className="flex items-center justify-center">
-            <ReportBadge name="loading.eth" />
+            <ReportBadge ensName={getExampleReportName()} />
           </div>
         </div>
 
         <div className="grid grid-cols-6 gap-x-6 py-5">
           <div className="flex items-center">
             <div className="flex items-center font-mono">
-              <pre>{"<ReportBadge onClick />"}</pre>
+              <pre>{"<ReportBadge \n   onClickOverride />"}</pre>
             </div>
           </div>
 
           <div className="flex items-center justify-center">
             <ReportBadge
-              name="lightwalker.eth"
-              data={getReportExampleData(Rating.pass)}
-              onClick={() => alert("Clicked")}
+              ensName={getExampleReportName(Rating.pass)}
+              data={getExampleReportData(Rating.pass)}
+              onClickOverride={() => alert("Clicked")}
```

Comment:
Please see my other comment about how the signature of the `onClickOverride` function should change to include the `ENSName` that was clicked on.

Please update the logic so that these "alert" calls are dynamic to show the specific name that was clicked on.